src/app/oss-routing.module.ts

```diff
@@ -183,16 +186,34 @@ const routes: Routes = [
             component: PreferencesComponent,
             data: { titleId: "preferences" },
           },
+          {
+            path: "security",
+            component: SecurityComponent,
+            data: { titleId: "security" },
+            children: [
+              { path: "", pathMatch: "full", redirectTo: "change-password" },
+              {
+                path: "change-password",
+                component: ChangePasswordComponent,
+                data: { titleId: "masterPassword" },
+              },
+              {
+                path: "two-factor",
+                component: TwoFactorSetupComponent,
+                data: { titleId: "twoStepLogin" },
+              },
+              {
+                path: "security-keys",
+                component: SecurityKeysComponent,
+                data: { titleId: "keys" },
+              },
+            ],
+          },
```

Comment:
This would be a good opportunity to extract these routes from the gigantic routing file similar to #1536

Reply:
Neat! Will update.

Reply:
Updated!